package.json

```diff
@@ -5,11 +5,11 @@
   "main": "dist/ink-es2015.js",
   "types": "ink.d.ts",
   "scripts": {
-    "test": "INK_TEST=dist jest && INK_TEST=legacy jest && jest --coverage && npm run-script test:remap",
-    "test:remap": "npm run-script test:remap:lcov && npm run-script test:remap:html && npm run-script test:remap:json",
-    "test:remap:lcov": "remap-istanbul --input coverage/coverage-final.json --output coverage/lcov.info --type lcovonly",
-    "test:remap:html": "remap-istanbul --input coverage/coverage-final.json --output coverage/lcov-report --type html",
-    "test:remap:json": "remap-istanbul --input coverage/coverage-final.json --output coverage/coverage-final.json",
+    "test": "npm run-script test:typescript && npm run-script test:javascript",
+    "test:typescript": "jest",
+    "test:javascript": "npm run-script build && npm run-script test:javascript:dist && npm run-script test:javascript:legacy",
```

Comment:
I don't think we should add the `build` step here, otherwise we'll run thebuild multiple times on the CI.

Reply:
On multiple occasions, I forgot to rebuild before running the tests, asking myself why my changes did nothing. For my own sanity, I added the build phase as a dependency of the test phase. But since JavaScript tests won't be used for debugging in the future, I will no longer pull my hair out.

I'm happy to revert that change!

Reply:
I generally start a build-watch in a separate console to avoid that, but I get the pain :)